### PR TITLE
Upgrade flow-bin from 0.95.1 to 0.105.2

### PR DESCRIPTION
**Pull request by aflexport**
Fix #82 function type issue.

### CODE REVIEW

File: src/test/tools.js [hunk 1]
```diff
@@ -41,3 +42,5 @@ export function mockLink<T>(formState: FormState<T>): FieldLink<T> {
     onBlur: jest.fn(),
   };
 }
+
+export const mockFn = (fn: any) => ((fn: any): JestMockT);
```
Comment (dmnd) [inline src/test/tools.js]: Perhaps we can update the `flow-typed` declarations for Jest instead? See https://github.com/flow-typed/flow-typed/blob/master/definitions/npm/jest_v24.x.x/flow_v0.104.x-/jest_v24.x.x.js#L1

Comment (aflexport) [reply src/test/tools.js]: I updated the declarations but it didn't work.
```javascript
export function mockLink<T>(formState: FormState<T>): FieldLink<T> {
  return {
    path: [],
    formState,
    onChange: jest.fn(),
    onBlur: jest.fn(),
  };
}
``` 
And I think `FieldLink<T>` already defined `onBlur`'s type, so calling `onBlur.mock` directly is not supposed to work whether we update `flow-bin` or not.

Now I'm doing it this way:
```javascript
const linkOnBlur: JestMockT = link.onBlur;
const newArrayTree = linkOnBlur.mock.calls[0][0];
```

Comment (aflexport) [reply src/test/tools.js]: I wrote this before you merged. Then I got into something else. LOL

Comment (dmnd) [reply src/test/tools.js]: 😆 haha ok, maybe I didn't read closely enough! CI passed and there were no `any`s so... 🤷‍♂

**Pull request merged**